packages/react/src/components/Coachmark/PositioningContainer/PositioningContainer.tsx

```diff
@@ -92,34 +92,46 @@ function usePositionState(
   };
 
   const updatePosition = (): void => {
-    const { offsetFromTarget, onPositioned } = props;
+    const { offsetFromTarget, onPositioned, target } = props;
     const hostElement = positionedHost.current;
     const positioningContainerElement = contentHost.current;
 
     if (hostElement && positioningContainerElement) {
       const currentProps: IPositionProps = { ...props } as IPositionProps;
       currentProps!.bounds = getCachedBounds();
       currentProps!.target = targetRef.current!;
-      if (document.body.contains(currentProps!.target as Node)) {
-        currentProps!.gapSpace = offsetFromTarget;
-        const newPositions: IPositionedData = positionElement(currentProps!, hostElement, positioningContainerElement);
-        // Set the new position only when the positions are not exists or one of the new positioningContainer positions
-        // are different. The position should not change if the position is within 2 decimal places.
+
+      if (target) {
+        // Check if the target is an Element or a MouseEvent and the document contains it
+        // or don't check anything else if the target is a Point or Rectangle
         if (
-          (!positions && newPositions) ||
-          (positions && newPositions && !arePositionsEqual(positions, newPositions) && positionAttempts.current < 5)
+          (!(target as Element).getBoundingClientRect && !(target as MouseEvent).preventDefault) ||
+          document.body.contains(currentProps!.target as Node)
```

Comment:
If we take `target` from `currentProps` then this could be 
```suggestion
          (!(target as Element).getBoundingClientRect && !(target as MouseEvent).preventDefault) ||
          document.body.contains(target as Node)
```